pages/1.11/monitoring/logging/access-component-logs/index.md

```diff
@@ -8,40 +8,38 @@ beta: true
 enterprise: true
 ---
 
-You can restrict user access to system and component logs. 
+You can restrict user access to system and component logs.
 
 Here is the [permission](/1.11/security/ent/perms-reference/) that is required to view the system and component logs:
 
 |     Permission string     | full | C | R | U | D |
 |----------------------------|------|---|---|---|---|
 | `dcos:adminrouter:ops:system-logs` <br>Controls access to [System logs API](/1.11/api/master-routes/#system).                                                                                                                                                                      | x    |   |   |   |   |
 
-**Prerequisites:** 
+**Prerequisites:**
 
 - DC/OS and DC/OS CLI are [installed](/1.11/installing/oss/) and you are logged in as a superuser.
 
 # Via the DC/OS GUI
 
 ### Create the Users and Grant Permission
 
-1.  Select **Organization** and choose **Users**. Select an existing or create a new user. 
-    
+1.  Select **Organization** and choose **Users**. Select an existing or create a new user.
+
     ![New user](/1.11/img/new-user-generic.png)
-    
-1.  From the **Permissions** tab, select **ADD PERMISSION**.
 
-    ![Add permission to user](/1.11/img/permission-user.png)
```

Comment:
it seems like the picture is functional in the 1.10 version, would it make sense to copy it from the 1.10 over here? @yishan-lin

Reply:
👍